regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -103,6 +103,7 @@ var CommentReviewView = Backbone.View.extend({
       var $elm = $(elm);
       var $text = $elm.find('input');
       var $select = $elm.find('select');
+      var $selectDropdown = $elm.find('.select-dropdown');
```

Comment:
I removed the line where I was using toggleInput incorrectly but didn't put a new solution in place yet. I added WIP to the PR title for now to indicate this isn't ready and I still have to figure it out.